test/unit/utils/version-spec.js

```
@@ -473,12 +473,19 @@ describe('Unit: Utils: version', function () {
         });
 
         it('resolves if node version isn\'t compatible with ghost node version range and GHOST_NODE_VERSION_CHECK is set', async function () {
-            this.timeout(5000);
-            this.slow(2000);
+            const packageJson = sinon.stub().resolves({
+                name: 'ghost',
+                versions: {
+                    '1.0.0': {}
+                }
+            })
+            const proxied = proxyquire(modulePath, {
```

ESLint

[error] Missing semicolon.

(semi)